site/_docs_1_0/httpproxy.md

```diff
@@ -990,6 +990,31 @@ spec:
       weight: 20
 ```
 
+## Upstream Validation
+
+When defining upstream services on a route, it's possible to configure the connection from Envoy to the backend endpoint to communicate over TLS.
+A CA certificate can be supplied to verify the backend endpoint contains a valid certificate.
```

Comment:
I think we can improve this by explicitly noting that CA bundles are stores in secrets. How about something like this:

```
A CA certificate bundle for the backend service should be supplied in a Kubernetes Secret.
The referenced Secret must be of type "Opaque" (this type of Secret is generated by the
[kubectl create secret generic](https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#-em-secret-generic-em-) command), 
and have a data key named `ca.crt`. This data value must be a PEM-encoded certificate bundle.
```